android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -94,6 +94,16 @@ private Boolean isTablet() {
     return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
   }
 
+  private float width() {
+    Configuration config = getReactApplicationContext().getResources().getConfiguration();
+    return config.screenWidthDp;
+  }
+
+  private float height() {
+    Configuration config = getReactApplicationContext().getResources().getConfiguration();
+    return config.screenHeightDp;
+  }
+  
```

Comment:
Same remarks for Android (names are too generic + let's try to merge this internally)